File: TsogosunProfileAdmin/MSProfileAdmin/Model/Dtos/PatronDrawDto.cs
using System;

namespace tsogosun.com.MSProfileAdmin.Model.Dtos
{
    public class PatronDrawDto
    {
        public string PromotionName { get; set; }
        public DateTime? DrawDate { get; set; }
        public int? IngenicoTickets { get; set; }
        public int? FreeIssueTickets { get; set; }
        public int? FreeIssueBonusTickets { get; set; }
        public int? PointsTickets { get; set; }
        public int? PointsTransactionTickets { get; set; }
        public int? PointsToNextTicket { get; set; }
        public int? CollectableTickets { get; set; }
        public int? CollectedTickets { get; set; }
        public DateTime? DateCollected { get; set; }
        public string DisplayOn { get; set; }
        public string PromotionsStatus { get; set; }
        public string TicketStatus { get; set; }
       
    }
}

File: TsogosunProfileAdminWeb/src/app/model/Dtos/profile-admin/patronDrawDto.ts
export interface PatronDrawDto {
    promotionName?: string | undefined;
    drawDate: Date;
    ingenicoTickets: number;
    freeIssueTickets: number;
    freeIssueBonusTickets: number;
    pointsTickets: number;
    pointsTransactionTickets: number;
    pointsToNextTicket: number;
    collectableTickets: number;
    collectedTickets: number;
    dateCollected?: Date | undefined;
    displayOn?: string | undefined;
    promotionsStatus?: string | undefined;
    ticketStatus?: string | undefined;
}
